packages/platform/types/slots/slots-2024-09-04/inputs/get-slots.input.ts

```diff
@@ -89,26 +88,6 @@ export class GetAvailableSlotsInput_2024_09_04 {
     example: "abc123def456",
   })
   bookingUidToReschedule?: string;
-
-  @IsString()
-  @IsOptional()
-  @ApiHideProperty()
-  teamMemberEmail?: string;
-
-  @IsNumber()
-  @IsOptional()
-  @ApiHideProperty()
-  routingFormResponseId?: number;
-
-  @IsArray()
-  @IsOptional()
-  @ApiHideProperty()
-  routedTeamMemberIds?: number[];
-
-  @IsBoolean()
-  @IsOptional()
-  @ApiHideProperty()
-  skipContactOwner?: boolean;
```

Comment:
removing the props that aren't documented and shouldn't be used for the get slots input.